graphql2/graphqlapp/alert.go

```diff
@@ -403,6 +403,17 @@ func (m *Mutation) CreateAlert(ctx context.Context, input graphql2.CreateAlertIn
 		a.Dedup = alert.NewUserDedup(*input.Dedup)
 	}
 
+	if input.Meta != nil {
+		alertMeta := alert.AlertMetaData{}
+		for _, v := range input.Meta {
+			alertMeta[v.Key] = v.Value
+		}
+		a.Meta = alert.AlertMeta{
+			Type:        alert.TypeAlertMetaV1,
+			AlertMetaV1: alertMeta,
+		}
+	}
```

Comment:
Use `withContextTx` and create the alert, and then if this is `!= nil` call the AlertStore method directly (part of separating Meta from the base Alert struct)